### PR TITLE
Xml templates

### PR DESCRIPTION
**Pull request by yhalapup**

### CODE REVIEW

File: itaplay/itaplay/static/styles/main.css [hunk 1]
```diff
@@ -193,4 +198,5 @@ form .progress {
 .image_current_clip {
   width: 640px;
   height: 480px;
+>>>>>>> origin/dev
```
Comment (lhalam) [inline itaplay/itaplay/static/styles/main.css]: delete this line

File: itaplay/itaplay/static/js/app/templates/controllers/add-template-controller.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+function AddTemplateController($scope, Upload, $timeout, $location) {
+
+    $scope.init = function() {
+        $scope.uploadXml = function(file) {
```
Comment (ypyrko) [inline itaplay/itaplay/static/js/app/templates/controllers/add-template-controller.js]: why you call uploading on page init?

File: itaplay/itaplay/static/js/app/views/header.html [hunk 3]
```diff
@@ -23,6 +23,6 @@
   <md-button href="#allclips" class="groupX middle">clips</md-button>
   <md-button class="groupX middle">players</md-button>
   <md-button class="groupX middle">messages</md-button>
-  <md-button class="groupX right">templates</md-button> 
+  <md-button href="#templates" class="groupX right">templates</md-button>
```
Comment (ypyrko) [inline itaplay/itaplay/static/js/app/views/header.html]: href="#templates" -> href="#/templates"

**Comment by vova-kul** [comment]
Can one of the admins verify this patch?

**Comment by vova-kul** [comment]
Can one of the admins verify this patch?

**Comment by vova-kul** [comment]
ok test

**Comment by vova-kul** [comment]
ok to test

**Pull request merged**